### PR TITLE
Add FY23Q2 OKRs for the IAM team

### PR DESCRIPTION
**Pull request by ryphil**

### CODE REVIEW

File: content/strategy-goals/strategy/cloud/iam-admin/index.md
Comment (bcreager) [inline content/strategy-goals/strategy/cloud/iam-admin/index.md]: Do we use customer names in the handbook?

File: content/strategy-goals/strategy/cloud/iam-admin/index.md
Comment (bcreager) [inline content/strategy-goals/strategy/cloud/iam-admin/index.md]: Using numbers here makes is feel like they are in priority order, which I don't think is the intent.  Could we name the problem areas or maybe just make a bulleted list so we don't infer priority?

Comment (ryphil) [reply content/strategy-goals/strategy/cloud/iam-admin/index.md]: Agreed - I removed the number and replaced with a short title of the problem area. How does this feel?

**Review by bcreager — APPROVED** [review]

**Review by quinnkeast — APPROVED** [review]

File: content/strategy-goals/strategy/cloud/iam-admin/index.md
Comment (unknwon) [inline content/strategy-goals/strategy/cloud/iam-admin/index.md:7]: qq: These are sub sections of "Objective and Key Results" but why is they are named as "Problem Area:" instead of "Objective:"?

File: content/strategy-goals/strategy/cloud/iam-admin/index.md
Comment (unknwon) [inline content/strategy-goals/strategy/cloud/iam-admin/index.md]: nit: other "Problem Area" use Title case but this one is using sentence case (we should use the latter according to our handbook, but being consistent is just as good enough).

File: content/strategy-goals/strategy/cloud/iam-admin/index.md
Comment (unknwon) [inline content/strategy-goals/strategy/cloud/iam-admin/index.md:45]: Does the order/number of the key results matter? Looking at other key results their order seems making sense to me in the order they are define. But the order here which would make sense to me is 3, 1, 2.

File: content/strategy-goals/strategy/cloud/iam-admin/index.md
Comment (ryphil) [reply content/strategy-goals/strategy/cloud/iam-admin/index.md]: Fixed in next commit!

File: content/strategy-goals/strategy/cloud/iam-admin/index.md
Comment (ryphil) [reply content/strategy-goals/strategy/cloud/iam-admin/index.md:45]: @unknwon - The ordering is closer to "How it happened", rather than any amount of importance. The #1 is already done and was *very* hacky solution because these customers were blocked on a production deployment. Then we started to build the plan for #2, and realized there are a few other short term fixes that would make our exiting solution less hacky, while still providing short term value to customers wanting this solution.

 So ultimately, the goal in Q2 is to provide a few short term fixes for customers and gather agreement on a long-term product improvement.

File: content/strategy-goals/strategy/cloud/iam-admin/index.md
Comment (ryphil) [reply content/strategy-goals/strategy/cloud/iam-admin/index.md:7]: @unknwon - Good question and callout. 

There are a few reasons: 
- Since we are fairly new, the various areas are at different levels of scoping. So we felt defining as problem areas was a good normalizer for areas we do not have any outcomes defined yet. 
- It brings the user voice problem into the planning process so we make sure our plans for the quarter are tied to user problems
- It lets our team be in the driver seat of defining the best solution, instead of embedding a solution in results.

Comment (ryphil) [reply content/strategy-goals/strategy/cloud/iam-admin/index.md:7]: @unknwon - I'm going to merge this and we can update if you feel we should adjust the heading.

File: content/strategy-goals/strategy/cloud/iam-admin/index.md
Comment (unknwon) [reply content/strategy-goals/strategy/cloud/iam-admin/index.md:45]: Makes sense, thanks!

File: content/strategy-goals/strategy/cloud/iam-admin/index.md
Comment (unknwon) [reply content/strategy-goals/strategy/cloud/iam-admin/index.md:7]: @ryphil thanks, no need to update the headings as was literally a question :)

Though just speaking on the matter, "solving problem area XYZ" could just be our objectives, if that makes sense to be an "objective" at all.

**Pull request merged**
